traffic_ops/traffic_ops_golang/server/detail_test.go

```diff
@@ -93,8 +94,6 @@ func TestGetDetailServers(t *testing.T) {
 	serviceNetmask := util.StrPtr("")
 	serviceInterface := util.StrPtr("")
 	serviceMtu := util.StrPtr("")
-	//routerHostName := util.StrPtr("")
-	//routerPort := util.StrPtr("")
```

Comment:
why were these commented out?

Reply:
Because both these fields are not part of [ServerDetail](https://github.com/apache/trafficcontrol/blob/master/lib/go-tc/servers.go#L1304){}

Reply:
Alright, that checks out. They should probably just be removed instead of merely commented-out, in that case.